src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.ts

```diff
@@ -243,6 +250,16 @@ export class OsdListComponent extends ListWithDetails implements OnInit {
   }
 
   ngOnInit() {
+    this.tableActions.unshift({
+      name: this.actionLabels.CREATE,
```

Comment:
Why moved it to  `ngOnInit()` ?

Reply:
@avanthakkar The recommended practice is to is put as little logic into the constructor as possible (other than injecting dependencies or other critical init logic that needs to be there).
[Almost all your initialization logic should be placed in `ngOnInit`]([url](https://www.geeksforgeeks.org/what-is-the-difference-between-constructor-and-ngoninit-in-angularjs/)).